plugin/pkg/admission/limitranger/admission.go

```diff
@@ -417,6 +417,12 @@ func (d *DefaultLimitRangerActions) ValidateLimit(limitRange *corev1.LimitRange,
 // SupportsAttributes ignores all calls that do not deal with pod resources or storage requests (PVCs).
 // Also ignores any call that has a subresource defined.
 func (d *DefaultLimitRangerActions) SupportsAttributes(a admission.Attributes) bool {
+	// Handle the special case for in-place pod vertical scaling
```

Comment:
why do we call it "special case" in comments? Maybe explain what so special about it

Reply:
Done.